### PR TITLE
eslint chores

### PR DESCRIPTION
**Pull request by tylim88**
1. remove redundant eslint config
2. move package.json eslint config to eslintrc (eslint ignore config in package.json if eslintrc exist)

effect:

from this
image

to this
image

### CODE REVIEW

**Review by HenryHengZJ — DISMISSED** [review]
thanks for cleaning that up! Would like to have prettier and eslintrc properly configured and applied to all 3 packages one day

**Comment by tylim88** [comment]
wait, few more changes

**Comment by tylim88** [comment]
accidentally pushed prettified contractDialog, code logic remain unchanged, just formatted

**Comment by HenryHengZJ** [comment]
to this branch. Do you not have problem when running `yarn build` on this branch? I had some ui problems there

**Comment by tylim88** [comment]
> to this branch. Do you not have problem when running `yarn build` on this branch? I had some ui problems there

yarn build did cause issue before this commit https://github.com/Outerbridgeio/Outerbridge/commit/179389b6dc57bb78527ab2b94daf790695d899f8, and this commit fix it

but now commit https://github.com/Outerbridgeio/Outerbridge/pull/86/commits/f31c727a9cd1811bd9f5d9edae23688fecb1ce0f causing some eslint prettier issue and I am fixing it

**Comment by HenryHengZJ** [comment]
Commit hooks is better to be added after we have prettier eslint configured for all 3 packages

File: packages/ui/.eslintrc [hunk 1]
```diff
@@ -35,21 +35,16 @@
...
+        "import/no-anonymous-default-export": 0,
         "no-unused-vars": [
             1,
             {
                 "ignoreRestSiblings": false
             }
         ],
         "prettier/prettier": [
-            2,
+            "error",
```
Comment (HenryHengZJ) [inline packages/ui/.eslintrc:46]: what's this `error` for?

**Comment by HenryHengZJ** [comment]
@tylim88 not sure if you have this but My VSCode is complaining about babel 
image
To solve this I had to add https://stackoverflow.com/a/72625130 `requireConfigFile: false`

File: packages/ui/.eslintrc (hunk 1 above)
Comment (tylim88) [reply packages/ui/.eslintrc:46]: it is an alias, 

0 : off
1: warning
2: error

I use English word to make it readable

Comment (tylim88) [reply packages/ui/.eslintrc:46]: soon will change every number to english word

Comment (tylim88) [reply packages/ui/.eslintrc:46]: basically what it does is, linter will raise error if any of prettier rule is violated,

**Comment by tylim88** [comment]
> @tylim88 not sure if you have this but My VSCode is complaining about babel image To solve this I had to add https://stackoverflow.com/a/72625130 `requireConfigFile: false`

I don't have this issue but I can add it

**Review by HenryHengZJ — APPROVED** [review]

**Pull request merged**